baseplate/sidecars/trace_publisher.py

```diff
@@ -128,6 +153,12 @@ def publish_traces() -> None:
         default="main",
         help="name of trace queue / publisher config (default: main)",
     )
+    arg_parser.add_argument(
+        "--queue-type",
+        default=QueueType.POSIX.value,
+        choices=[qt.value for qt in QueueType],
+        help="whether to use an in-memory queue or a posix queue",
```

Comment:
Best not to explicitly state the queue types as they could change. Use a more generic message.